### PR TITLE
Basic reporting of upcoming payments

### PR DESCRIPTION
**Pull request by petrblaho**
This is just a function for returning these upcoming payments, it does not print anything to output.

### CODE REVIEW

File: zonkylla/core/payment_plan.py [hunk 1]
```diff
@@ -5,14 +5,24 @@
 
 '''Payment plan module'''
 
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+
+
```
Comment (celestian) [inline zonkylla/core/payment_plan.py]: Please, remove one empty line. I checked, it is ok from pep8 point of view.

File: zonkylla/core/payment_plan.py [hunk 2]
```diff
@@ -5,14 +5,24 @@
...
+from .utils import iso2datetime
+
 
 class PaymentPlan():
     '''Payment plan class'''
 
-    def __init__(self, amount, interest_rate, term_in_months):
+    def __init__(self, amount, interest_rate, term_in_months, start_date=None):
```
Comment (celestian) [inline zonkylla/core/payment_plan.py]: Could you please refactor those parameters to just one (which will be object with duck-typing property how we discussed?)

File: zonkylla/core/payment_plan.py (hunk 1 above)
Comment (petrblaho) [reply zonkylla/core/payment_plan.py]: Done.

File: zonkylla/core/payment_plan.py (hunk 2 above)
Comment (petrblaho) [reply zonkylla/core/payment_plan.py]: Done.

**Comment by petrblaho** [comment]
Updated and rebased.

**Pull request merged**